### PR TITLE
Allow to upgrade Kubeapps through the Dashboard

### LINKED ISSUE #463: Can't install multiple Kubeapps instances via Helm Chart
During installation, the Helm chart allows you to select the namespace you want to deploy Kubeapps to, which works fine for the first installation in the cluster but it fails if there is already an existing kubeapps instance installed.

This is because the chart tries to install again cluster-wide resources resulting in for example

```
Error: release kubeapps failed: customresourcedefinitions.apiextensions.k8s.io "apprepositories.kubeapps.com" already exists
```

The same issue applies to cluster-roles.

We should extend our Helm chart to skip these resources if they already exist.

### LINKED ISSUE #465: Add the Bitnami chart repo to the list of repos to add by default
The Bitnami chart repo contains a set of charts that are known to be continuously tested and up to date. We should add it to the list of repos that Kubeapps adds by default

### PR DESCRIPTION
**Pull request by andresmgot**
Fixes #465: Install `bitnami` repo by default

### CODE REVIEW

**Comment by andresmgot** [comment]
Added that as a `tip` @arapulido

**Comment by arapulido** [comment]
Thanks @andresmgot I would also add that tip to the getting-started.md doc and the README.md of the repo as well

**Comment by andresmgot** [comment]
_Done_

Maybe we have the installation guide in too many places

File: chart/kubeapps/values.yaml [hunk 1]
```diff
@@ -23,6 +23,8 @@ apprepository:
     url: https://kubernetes-charts-incubator.storage.googleapis.com
   - name: svc-cat
     url: https://svc-catalog-charts.storage.googleapis.com
+  - name: bitnami
```
Comment (migmartri) [inline chart/kubeapps/values.yaml]: Can we add this in another PR? It does not seem to be related to the rest of the PR.

**Comment by migmartri** [comment]
Oops, just noticed that my comment makes more sense in the context of the PR itself. I still get lost in Github... :S 

https://github.com/kubeapps/kubeapps/issues/463#issuecomment-412154932

**Comment by prydonius** [comment]
Let's hold on this till we decide what the correct approach is.

**Comment by andresmgot** [comment]
I am going to revert the changes related to the cluster-wide objects and leave this PR for fixing #465 only.

File: chart/kubeapps/values.yaml (hunk 1 above)
Comment (migmartri) [reply chart/kubeapps/values.yaml]: Thank you!

**Comment by prydonius** [comment]
Can we add an explanation in the git commit, e.g. something about enabling upgrading the Kubeapps Helm chart from Kubeapps itself.
